src/wymeditor/core.js

```diff
@@ -534,9 +515,6 @@ jQuery.extend(WYMeditor, {
         // Path to the main JS files
         wym._options.basePath = wym._options.basePath ||
             WYMeditor._computeBasePath(wym._options.wymPath);
-        // Path to jQuery (for loading in pop-up dialogs)
-        wym._options.jQueryPath = wym._options.jQueryPath ||
-            WYMeditor._computeJqueryPath();
```

Comment:
@winhamwr thanks for checking. Removed the `jQueryPath` option and `._computeJqueryPath`. Updated the docs and changelog.